Makefile.docs

```diff
@@ -13,6 +13,7 @@ REPOSITORY ?=
 VERSION_NAME ?=
 
 API_DOCS_FOLDER := "$(DESTINATION_FOLDER)/api"
+SITE_DOCS_FOLDER := "$(DESTINATION_FOLDER)/public"
```

Comment:
Can you confirm me that for a given `gogit/v3`, the structure of the site will be:
```
gogit/v3/
    (root) -> generated by ci:docs-site
    api -> generated by ci:docs-api
```
Because I initially understood that the site would be served from `/public`, an `/api` sibling directory.